CMakeLists.txt

```diff
@@ -265,6 +265,10 @@ else()
                 COMPONENT devel)
         file(WRITE ${CMAKE_CURRENT_BINARY_DIR}/${PROJECT_NAME}Config.cmake
                    "include(\${CMAKE_CURRENT_LIST_DIR}/${PROJECT_NAME}Targets.cmake)\n")
+        if (NOT BUILD_SHARED_LIBS)
+            file(APPEND ${CMAKE_CURRENT_BINARY_DIR}/${PROJECT_NAME}Config.cmake
+                       "include(CMakeFindDependencyMacro)\nfind_dependency(Threads)\n")
```

Comment:
I suppose we can directly patch this file https://github.com/oneapi-src/oneTBB/blob/master/cmake/templates/TBBConfig.cmake.in as a template

Reply:
The template and these install instructions are not connected, actually. The template is generated using separate CMake script, but these instructions are for TBBConfig.cmake generation using, for example, `cpack` command. I think this is the right spot for the change, since the template is dedicated for shared library build.